backend/infrahub/core/node/__init__.py

```diff
@@ -272,6 +274,27 @@ async def handle_pool(self, db: InfrahubDatabase, attribute: BaseAttribute, erro
                 )
             )
 
+    async def handle_object_template(self, fields: dict, db: InfrahubDatabase) -> None:
+        """Fill the `fields` parameters with values from an object template if one is in use."""
+        object_template_field = fields.get("object_template")
+        if not object_template_field:
+            return
+
+        template: CoreObjectTemplate = await registry.manager.find_object(
+            db=db,
+            kind=self._schema.get_relationship(name="object_template").peer,
+            id=object_template_field.get("id"),
+            hfid=object_template_field.get("hfid"),
+            branch=self.get_branch_based_on_support_type(),
+        )
+
+        # Handle attributes, copy values from template
+        # Relationships handling in performed in GraphQL mutation to create nodes for relationships
+        for attribute in template._attributes:
```

Comment:
This looks like this would fail if we fail to find the specified template. I.e. if there's a typo in the template name or if the template doesn't exist. If that's the case I think we should raise a validation error here.